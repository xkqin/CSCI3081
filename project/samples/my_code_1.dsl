/* This program is just a copy of sample_1.dsl. 
You should write your FCAL program to multiply two
2-dimensional matrices and save it
in a file with this name. */

main () { 

  matrix l = matrix_read ( "./samples/my_code_1.data" ) ;
  matrix m = matrix_read ( "./samples/sample_8.data" ) ;
  matrix ans = l * m ;
  
  print ( l ) ;
  print ( m ) ;
  print ( ans ) ;
}



