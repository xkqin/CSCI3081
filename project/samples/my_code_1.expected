3 4
0  1  2  3  
4  5  6  7  
8  9  10  11  
4 5
1  2  3  4  5  
2  3  4  5  6  
3  4  5  6  7  
4  5  6  7  8  
3 5
20  26  32  38  44  
60  82  104  126  148  
100  138  176  214  252  
